vector/src/main/res/values/strings_login_v2.xml

```diff
@@ -43,7 +43,7 @@
     <string name="login_set_msisdn_title_2">Associate a phone number</string>
     <string name="login_set_msisdn_notice_2">Associate a phone number to optionally allow people you know to discover you.</string>
     <string name="login_set_msisdn_mandatory_notice_2">The server %s requires you to associate a phone number to create an account.</string>
-    <!-- %s will be replaced by the value of action_learn_more ("Learn more" in English)-->
+    <!-- %S will be replaced by the value of login_server_modular_learn_more ("Learn more" in English)-->
```

Comment:
I think you have been a bit fast with the "replace all" command :/

Reply:
I never use that command

Reply:
OK